Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -83,6 +84,52 @@ public void DeserializesObjectsWithIgnoreDefaults()
             Assert.IsNull(deserialized.color);
             Assert.IsNotNull(deserialized.canOffroad);
         }
+        
+        [TestMethod]
+        public void SerializesAndDeserializesWithCustomSerializers()
```

Comment:
Tests are cheap, you can write lots of them. This one test has 10 assertions. I'd rather see 10 tests, appropriately named with, (approximately) one assertion each.